packages/core/src/Stores/Helpers/chart-markers.js

```diff
@@ -112,6 +117,9 @@ const createTickMarkers = contract_info => {
         if (marker_config) {
             result.push(marker_config);
         }
+        if (is_only_ups_downs && !is_entry_spot && result.length > 2) {
```

Comment:
According to the new requirements we should hide previous label with tick info when the next marker appeared (for middle spots). When the exit_spot appeared we hide the last marker for middle spot.

Reply:
@kate-deriv please unite it into one condition before accumulator condition like this🙂:
```
if (is_only_ups_downs) {
    if (is_middle_spot) {
        const spot_className = marker_config.content_config.spot_className;
        marker_config.content_config.spot_className = `${spot_className} ${spot_className}--only-ups-downs-small`;
    }
    if (!is_entry_spot && is_preexit_spot && tick_stream.length > 2) {
        marker_config.content_config.is_value_hidden = true;
    }
}
```

Reply:
@maryia-deriv but I need to hide label of the **previous** spot **only after new one appeared**. Please, have a look to the pinned recording in the card or figma prototype in order to see the behaviour of the chart markers and labels🙇🏼‍♀️

Reply:
And in you suggestion you'll hide current spot label🙇🏼‍♀️

Reply:
Conclusion: after small discussion Maryia suggest a great solution, which was implemented.
```
if (is_only_ups_downs && is_middle_spot) {
            const spot_className = marker_config.content_config.spot_className;
            marker_config.content_config.spot_className = `${spot_className} ${spot_className}--only-ups-downs- 
            small`;
            if (!is_current_last_spot) {
                marker_config.content_config.is_value_hidden = true;
            }
}
```